crates/extensions/tedge_config_manager/Cargo.toml

```
@@ -0,0 +1,34 @@
+[package]
+name = "tedge_config_manager"
+description = "thin-edge Cumulocity extension adding support for configuration management"
```

Obviously we should get rid of "Cumulocity" :D

```suggestion
description = "thin-edge extension adding support for configuration management"
```